test/integration/integration_admin_test.h

```diff
@@ -2,18 +2,14 @@
 
 #include "common/json/json_loader.h"
 
-#include "test/integration/http_integration.h"
+#include "test/integration/http_protocol_integration.h"
 
 #include "gtest/gtest.h"
 
 namespace Envoy {
 
-class IntegrationAdminTest : public HttpIntegrationTest,
-                             public testing::TestWithParam<Network::Address::IpVersion> {
+class IntegrationAdminTest : public HttpProtocolIntegrationTest {
```

Comment:
This makes me happy :-)

one nit though - may be worth doing 
getProtocolTestParams({Http::CodecClient::Type::HTTP1, Http::CodecClient::Type::HTTP2}, {FakeHttpConnection::Type::HTTP1}) in INSTANTIATE_TEST_CASE_P - I don't think testing H2 upstream and H1 upstream matters for the admin handler, yeah?

Reply:
Yeah it was so easy! Sure will change, wasn't sure how to do that.